spec/core_functions/color/same.hrx

```diff
@@ -0,0 +1,244 @@
+<===> true/same_space/identical/input.scss
+@use 'sass:color';
+a {b: color.same(#abcdef, #abcdef)}
```

Comment:
Out of curiosity, is `color.same(#abc, #abcf)` covered by `true/same_space/none_and_zero` or by `true/different_space/no_none/input`?

Reply:
`#abcf` isn't a color. Do you mean `#abcdef`? Those would be different colors, so they'd be under `false/`.